src/main/java/graphql/nadel/engine/NadelExecutionStrategy.java

```diff
@@ -211,6 +223,19 @@ public NadelExecutionStrategy(List<Service> services,
         return resultNodes;
     }
 
+    private RootExecutionResultNode getSkippedServiceCallResult(NadelContext nadelContext, ExecutionStepInfo esi, ExecutionContext newExecutionContext, TransformationMetadata.NormalizedFieldAndError topLevelFieldError) {
+        HashMap<String, Object> stringObjectHashMap = new HashMap<>();
+        HashMap<String, Object> errorMap = new HashMap<>();
+        errorMap.put("message", topLevelFieldError.getError().getMessage());
+        stringObjectHashMap.put(esi.getFieldDefinition().getName(), null);
```

Comment:
Nit: move this line right below the definition, I think it reads better when you define the var and immediately use it. That way there's we don't have to juggle the two vars in our heads.